en/docs/index.md

```diff
@@ -40,8 +43,8 @@ template: templates/single-column.html
                         <i class="material-icons md-36">library_books</i>
                     </div>
                     <div class="card-content">
-                        <p class="title">Security Guideliness</p>
-                        <p class="hint">Read our guidelines to understand how to enforce security-first development and deployment practices.</p>
+                        <p class="title">Security Guidelines</p>
+                        <p class="hint">How to enforce security-first development and deployment practices</p>
```

Comment:
```suggestion
                        <p class="hint">Learn how to enforce security-first development and deployment practices</p>
```

Reply:
Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8